apps/web/app/api/comments/[id]/like/route.ts

```
@@ -0,0 +1,63 @@
+                const id = params.id
+                if (!id) return NextResponse.json({ error: 'Missing id' }, { status: 400 })
+
+                const { data: existing, error: fetchError } = await supabase
+                        .from('comments')
+                        .select('id, metadata')
+                        .eq('id', id)
+                        .single()
+
+                if (fetchError || !existing) return NextResponse.json({ error: 'Comment not found' }, { status: 404 })
+
+                const metadata = (existing.metadata as Record<string, any>) || {}
```

Biome (2.4.15)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)